gdk4-win32/src/auto/win32_hcursor.rs

```diff
@@ -0,0 +1,113 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files.git)
+// DO NOT EDIT
+
+use crate::Win32Display;
+use glib::object::Cast;
+use glib::object::IsA;
+use glib::object::ObjectType as ObjectType_;
+use glib::translate::*;
+use glib::StaticType;
+use glib::ToValue;
+use std::fmt;
+
+glib::wrapper! {
+    #[doc(alias = "GdkWin32HCursor")]
+    pub struct Win32HCursor(Object<ffi::GdkWin32HCursor, ffi::GdkWin32HCursorClass>);
+
+    match fn {
+        type_ => || ffi::gdk_win32_hcursor_get_type(),
+    }
+}
+
+impl Win32HCursor {
+    #[doc(alias = "gdk_win32_hcursor_new")]
+    pub fn new(display: &Win32Display, handle: win32::HCURSOR, destroyable: bool) -> Win32HCursor {
+        skip_assert_initialized!();
+        unsafe {
+            from_glib_full(ffi::gdk_win32_hcursor_new(
+                display.to_glib_none().0,
+                handle,
+                destroyable.into_glib(),
+            ))
+        }
+    }
+
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct [`Win32HCursor`] objects.
+    ///
+    /// This method returns an instance of [`Win32HCursorBuilder`](crate::builders::Win32HCursorBuilder) which can be used to create [`Win32HCursor`] objects.
+    pub fn builder() -> Win32HCursorBuilder {
+        Win32HCursorBuilder::default()
+    }
+
+    pub fn is_destroyable(&self) -> bool {
+        glib::ObjectExt::property(self, "destroyable")
+    }
+
+    pub fn display(&self) -> Option<gdk::Display> {
+        glib::ObjectExt::property(self, "display")
+    }
+
+    //pub fn handle(&self) -> /*Unimplemented*/Basic: Pointer {
```

Comment:
It seems like some property definition still needs a fixed type like what you did to others in the gir file.

Reply:
yes, I can patch it to say it's actually a HCURSOR (which maps to isize). Gdk implementation uses g_param_spec_pointer under the hood though, so this is not satisfied:
```
54   |         glib::ObjectExt::property(self, "handle")
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `for<'b> FromValue<'b>` is not implemented for `isize`
```

I guess it has to be manually implemented then. Other ideas?

Reply:
There are not that many of them, manually implementing it is fine.

Reply:
done. I need some solution for the filter callback stuff though... I don't get why it fails.

Reply:
I left the add_filter/remove_filter as TODO.